### DISCUSSION TITLE
8 個無程式碼 Machine Learning 平台　讓你把 AI 想法變成實際的 App！

### DISCUSSION THREAD

**Discussion by 32844583** [1101-GS3073-C/Python-Community-GS3073-C#514] [General]
### 上課教 teachable mechine 之外的其他類似平台
在過去的十年中，許多無程式人工智能（AI）平台湧現，使越來越多的公司能夠使用AI構建更智能的軟體。但是，由於機器學習的快速發展和尋找該領域專業人才的困難，許多人發現實施這些平台很困難。無程式可視化拖放工具的出現幫助解決了數據科學家的短缺，並使非技術人員更容易使用AI技術。
https://www.appcoda.com.tw/no-code-machine-learning-platforms/

**Comment by brian431** [comment]
這些軟體讓原本一般人難以理解或接觸的機器學習變得更加地平易近人，實在是非常實用的工具。

**Comment by NcuJerryKao** [comment]
不用打程式也能學ml 
對於看到程式碼有Bug就會頭痛 套件安裝有問題尚在解決的我來說 amazing

**Comment by gnsJhenJie** [comment]
課程提到的Azure Custom Vision也可以算是No-code或Low-code的ML平台

**Comment by skyling0299** [comment]
一直想到自動補碼系統

**Comment by williamyen0623** [comment]
無程式碼的開發工具大大增加社會生產力，讓那些有想法卻沒有基礎技術背景的人才能夠發揮才能

**Comment by limitx0** [comment]
感覺都是很有趣的平台，想去玩玩看

**Comment by Chenszuhong** [comment]
這個主題真是不錯的想法 感覺都很有趣而且很帥 想要知道更多能延生應用的地方

**Comment by Aerofe** [comment]
這些平台對那些初學者來說是個很大的幫助！

**Comment by awehao** [comment]
好想試試歐，程式的開發是一個可以不斷創新的事
